### PR TITLE
k8s-1.17: Fix non interactive dnf install -y

### PR DESCRIPTION
**Pull request by qinqon**
Signed-off-by: Quique Llorente <ellorent@redhat.com>

### CODE REVIEW

File: cluster-provision/k8s/1.17/provision.sh [hunk 1]
```diff
@@ -68,7 +68,7 @@ name=Kubernetes
 baseurl=http://yum.kubernetes.io/repos/kubernetes-el7-x86_64
```
Comment (ormergi) [inline cluster-provision/k8s/1.17/provision.sh]: try this url https://packages.cloud.google.com/yum/repos/kubernetes-el7-x86_64
before you change gpg check

**Comment by qinqon** [comment]
@dhiller I have tried it at kubernetes-nmstate jobs it does not work, also by default repogpgcheck is 0 if you don't specify so it's not that bad also we review the content of .repo file.

**Pull request merged**